ci-operator/step-registry/gcp/provision/xpn-min-perm-cco-manual-serviceaccount/gcp-provision-xpn-min-perm-cco-manual-serviceaccount-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: gcp-provision-xpn-min-perm-cco-manual-serviceaccount
```

Comment:
Like `AWS_CCOCTL_USE_MINIMAL_PERMISSIONS`, can we introduce a primary switch like `GCP_XPN_CCO_MANUAL_USE_MINIMAL_PERMISSIONS`, so it would be helpful while reading the job config and the workflow. WDYT?